tests/gis_tests/geos_tests/test_geos.py

```diff
@@ -227,7 +227,7 @@ def test_points(self):
             self.assertEqual(pnt.geom_typeid, 0)
             self.assertEqual(p.x, pnt.x)
             self.assertEqual(p.y, pnt.y)
-            self.assertEqual(True, pnt == fromstr(p.wkt))
+            self.assertEqual(pnt, fromstr(p.wkt))
             self.assertEqual(False, pnt == prev)
```

Comment:
assertEqual will test `__eq__` and assertNotEqual will test `__ne__`. Since the `GEOSGeometry` class appears to have a straight forward `__ne__` method, that maybe that doesn't matter 

        def __ne__(self, other):
            "The not equals operator."
            return not (self == other)

https://github.com/django/django/blob/master/django/contrib/gis/geos/geometry.py#L180

However, switching to assertNotEqual, does change the meaning of the test slightly, so I left them as they were.